# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).

## [Unreleased](https://github.com/RCIC-UCI-Public/biotools-admix/compare/2.0...HEAD)

### Commits

- add source for bustools 0.44.1 [`6f8b1b9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6f8b1b94120f367d5c6af880aaf9649eb4d02cbc)
- restore old bustools per #10858. add new version [`4e7fc0e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4e7fc0e2946c71e603e05a0a869e4c5abd2976d8)
- manifest is inherited from packages.yaml by each set [`6526bf7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6526bf7ce74ac4e54a675607d828a84931acd42c)
- rm fossils [`7ab6444`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7ab64440500602d18e8904128fa8d98a0eda3afc)
- use sed to update version of removed dependency in a remote maven repository [`a0b82c4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a0b82c4cc5549af297f28f9e7def1262cf769dff)
- add kallisto earlier version. [`80eb409`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/80eb409fba4d518a3322eab3adbe527f1a406b6f)
- reinstate freebayes, unicycler and its dependencies. [`46baac4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/46baac4596f61a5501f0254c5d70706db8f225c4)
- rm unused version of bowtie2 [`eff69cc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/eff69ccfc1a5bb8f1001bf7d8f81a03eb6dca92a)
- use cmake module directly [`92388ef`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/92388ef6fb3fbb74ff0b78f8b8c30447117a2e17)
- picard_tools 2.27.x does not build any longer (removed dependency in a remote maven repository causes a build failure). [`993569c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/993569ca6d313ea818ea9ba632a955ccf2956bfa)
- update htslib and bcftools to 1.21. Add filter for requires as [`c78f9c8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c78f9c83f00e93f981e402fb4384a8f84c0cca6f)
- latest kalisto needs to be compiled with large indexes enabled [`90887b2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/90887b27921049ad7f3af157961ab339aa6e9bdf)
- update to the latest kallisto and minimap2 [`b540903`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b540903a28c15fd22eb76d473c3b35f96a029b51)
- add temp dirs resulted form using wget and pip build commands [`4b8acd0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4b8acd09e9bfd9988bbde4bfc5f1ffb664dd633a)
- update CHANGELOG [`153a3c1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/153a3c19979f797b5aefc275fc62a44949bf9e18)
- rm  lz4 version [`486d51d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/486d51d0a558786c15ccee83f809847f54f4beb0)
- archive unsused packages. [`9d56b3a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9d56b3a7dd92b363689ce5f036658438f833f01b)
- neous file [`d94512b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d94512b83f1a27c91eec3188b8dfe94dab9ed85c)
- Setting Release-Version v2.25.0 [`3686362`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3686362bae6f7742f190edca3a5465840df1ef95)
- Setting Release-Version v2.25.0 [`8ec7d2a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8ec7d2a4df4c342bdbb7589744ceca212da0997a)
- content is provided by RpmExtras macro in rpm.yaml [`8b19bc9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8b19bc957372f15f8b3c900d0722d0c9dd63166b)
- rename sets and versions files for esiser tracking [`f125093`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f125093f47bb771060e7ca6359afd9c7df31992a)
- add auto-changelog generated CHANGELOG.md [`ba518ab`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ba518abe99e1d8ec638bdb7e36265bfd4dd8f605)

## [2.0](https://github.com/RCIC-UCI-Public/biotools-admix/compare/1.0...2.0) - 2023-07-14

### Commits

- disable sra-tools prior to v.3. Bad usage for version 2.* [`2442a44`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2442a444a63332768040cb419906525916f4ba80)
- disable unused racon, prereq for unused unicycler [`0eb08c3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0eb08c396b59e60d717c10dbcefec1e82809e568)
- ble older unused pilon, minimap2, unicycler, meme [`040bd26`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/040bd26d64f29cdefbb241d5056aa4aa6b967b0c)
- disable bamtools and its prereq jsoncpp, unused [`4ce768f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4ce768ff56c96402f0de74f69618fb97c1a68439)
- no cufflinks [`9909089`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/990908986cf057fb7f18dfbac2f1b587df71b127)
- deal with annobin configuration [`009ae2c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/009ae2c0b7084c12e2b49b3410cdc5c8500fd4db)
- configure for jellyfish.yaml fails with possibly undefined macro: AC_LIB_RPATH [`70ddba8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/70ddba88337f6f30bc0cf52fee4c93a05f5a1d24)
- Updated version of bowtie2 [`8f11cd8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8f11cd8533c87b5e75337b7d9f8c24dfb9d51f28)
- add sra-tools 3.0.0. Has improved fasterq-dump [`95b1c40`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/95b1c409416d8e369c0cd3d3740c17f91c5a007f)
- separation of options into --root and --prefix disables building of underlying [`4a9ee52`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4a9ee524fac8d4b3f98f07b27f464077071aa41c)
- update to picard-tools was on opt1  branch and unmerged. [`545ec6a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/545ec6a632b1b2edf979e18169adef08ccab9904)
- comment out unused rrbgen [`26a8d70`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/26a8d706aa59c6e96a1c018ffb9db429dd45f18e)
- comment out rrbgen, it is a prereq for unused stitch [`5f6c323`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5f6c323218a35062c71a84aa866dccef4dffbc8d)
- Don't bootstrap in set-py2. It uses the same bootstrapped versions as the base set [`937a7ec`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/937a7ecd336ccb13bd86bf00f6edf7b5a7146fa0)
- set bootstrap: to empty in set-latest. [`d92edfc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d92edfc6988faf05e7b0044b24ff8c79cb5fcf70)
- account for difference in tarball naming of older and newer versions of star [`fe8c65d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fe8c65d1a5b192bb5bfebe5c7b0743a0949bcf1f)
- Fixed upload of libdeflate in google drive ... new hash. [`2fff95d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2fff95d3b4f5a4407eeccbfa2be275ee2ffeb36b)
- Another configure weirding...need to unset CPPFLAGS prior to python setup.py. With is set, [`50fde2c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/50fde2c2230c20fa992717c0ff4946eaf5e80de2)
- ncbi-ngs, ncbi-vdb needed to compile hisat2 in the genomics admix [`f2656b5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f2656b5e6cbe96cd2bd3aa843ccec9e28ca760bc)
- move deeptools and cutadapt into their own app directory instead of [`1b2c2e2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1b2c2e24df9d062ab3462df123ea2c5895dc7083)
- add new sources to google drive [`daacda1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/daacda1e7207e07c4c134c402e683ad72def5afd)
- add latest packages for centos8 [`ab0a919`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ab0a9194611ffde367bb01b70e303ae5e1c7f813)
- update per changes to yaml files for source, pyversion, etc.. [`8fef622`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8fef6225d398bac3837ef10dba6f5db020b2745f)
- rm unsued minimap2 version. [`fe6b2f0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fe6b2f0d7f040b249754dea940d0a43977e8e0ea)
- rename set and version for python2 build [`12861d0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/12861d0529fe57a4cb8f67dcfe175dc0fb8c8298)
- use version and pymajor variables. this allows to use a single yaml [`81de2e0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/81de2e0b68964265a438d5febc73094b65fe60a8)
- set version and pymajor variables in version*yaml files and [`dd11d00`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/dd11d001fdabf30ea8875ab61f2cb2184fd28381)
- use default naming schema for src_tarball [`2e5d416`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2e5d416ffd72afd9efc91505023937521d6d028c)
- do not add htslib.mk htslib_vars.mk to the install directory. [`943996b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/943996b4d86f6d34e18754d611fa7de26dc925a1)
- newer distros  compilation fails with a latest (above v9 gcc) [`75e7cd0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/75e7cd0dad5849cfab12d3e6abe984967bcb2262)
- update vendor source for multiple versions, unset MAKEFLAGS in build [`851e1b8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/851e1b82521cd5721893a3a7ee30dcced5d8dd48)
- update vendor source for multiple versions [`3b7aa7b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3b7aa7b913187de972741f05c8bde2ebf2e19aa7)
- add --skip-npm option to a newer version build. [`615dbe4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/615dbe45a6755c6fe1c96b44a0c0b128e5801293)
- update vendor source pre new distro. [`8fd18a1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8fd18a17a947d3aeafd20ae164c98b1a96a4545e)
- update download link per new hierarchy. [`8b3795f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8b3795fdda8b0e09d108d8a02f418a155d859872)
- add a few more libraries to ignore in requires. [`48ce789`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/48ce78907ec25867b1759c83ae97691560282387)
- newer java openjdk do not provide "jre = " but provide "jre-" [`78b8c3c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/78b8c3c42d9e1fe616cf792035c443163dcd8d3d)
- newer version of fastp depends on 2 additioal libraries whihc come [`84bf5f6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/84bf5f674b0c51b5c79f07dac3fc2554eeb3fd36)
- cufflinks depends on boost where older version of boost had include/boost/tr1/* [`1a75575`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1a755757ee169183ef51532f635ac1b904f327c1)
- ideveloeprs do not version their distros. hae to go by year so that [`8f17f33`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8f17f33de7998fec4d6beeb01355bbac43ce93ed)
- use hash for multipel versions [`6b1d89a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6b1d89a79368845004b2b05b1f1722eaaee9a2c9)
- accommodate for patches releases [`ded211f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ded211f72cea846082f52cafab49de45c4c0cce9)
- source distro name change i late version. [`6446edf`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6446edf911318c958b9dad03cec734b86296ad3a)
- user foundaiton module with version [`858d5f7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/858d5f7abafe47f6ddb261810a888d3f546f90e1)
- one digit was flipped in checksum, fixed [`9cd8f1d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9cd8f1d5b3ef9eea4a40b27b6471991ca7b8563a)
- Add intel_tbb [`1462dc4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1462dc4b3ac507b01b4f2db3c1cfcdb05f21ad25)
- install jar file [`a3cf1f2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a3cf1f27fc4702742bd28c47752c0bc167347170)
- unset MAKEFLAGS [`d560427`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d5604273df45af33b74977912b388b4010d43c88)
- unset MAKEFLAGS [`c461cd0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c461cd01e78f55aaadd4f0f93cf8024f7e59f693)
- add src_tarball [`412940c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/412940c76ab2bb949daeed700cc8ed2df25edbd5)
- Don't build id links for minimap2  (both versions include same binary libraries) [`8c4f48e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8c4f48e8f867489864ef52e518f5153c27efbb2b)
- Remove perl requirements that are not explicit in trinity [`bbeb53e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/bbeb53e9602442e8a0d912e3f3267acff59fc9e5)
- System packages that are not available on EL8 (lz4-static, perl-Package-Constants) are not requested on EL8. [`0d08ac8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0d08ac8b8e0efed16ea2cbdd8b421a169ff1c268)
- Build cufflinks under EL8. Requires specific commit from cufflinks github. [`ed263bc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ed263bc79cbdc80dfcb7331b49217f8238326510)
- Build bcftools with annobin turned off on EL8 (and later) [`29a270e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/29a270e4f1763c4c5b32e0017df93c1d9c5cc0ee)
- Sambamba build with updated liblz4.a (built in buildtools) [`6f26821`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6f26821ac9205c590ec690590faca70449ed5128)
- add minimap2-2.23 to google drive [`42a65da`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/42a65da13b5e5126252b600932b1d46d420f6de0)
- change install path and module settings so that [`10bd6ca`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/10bd6ca9c5660080c25f6d4d5390b7425871864a)
- gsl is in site versions [`868cfe6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/868cfe60ff3c8128445a8868457c39360c61dd9c)
- use site versions [`5c29123`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5c29123d0cebaf5c724ee27da3657e61d8651c4b)
- in preparation to use common versions file [`3b0105d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3b0105d5927907d333c53406be600ce17c461153)
- add .rpmmacros [`26fcf49`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/26fcf49dba08b947154f2bc1c772fce77a65f8a2)
- delete extra definitions [`1e1b5a6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1e1b5a6913cd8a97a50d98be386aad42a8252810)
- sra tools fix typo gcc. -&gt; gcc/   in required modules [`f7d38eb`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f7d38ebd6b47560a27844718112e3f180c703a85)
- update naming schema [`801a23c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/801a23cf17d47bb597525631da3b8300a0fe2fb0)
- verify configure and its arguments [`d50a9b5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d50a9b5195172421163352a275843e9777e99e06)
- moving to set based build [`49c2dca`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/49c2dca9adb2db8899d7a7640f9ebca8ded10ff7)
- add breseq as requested in ticket #5850 [`9aa1bfc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9aa1bfcd5bb08dce9fb4b2dd612af83248f8d083)
- rm empty strings [`54e916f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/54e916f1e765b12e875ce032ff8fdf6a835b303b)
- initial cleanup for opt1 changes [`5580476`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5580476f114708cb523e74a18e5d59aa0e808ba1)
- add filters for perl and dependency on perl. [`1d6caf1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1d6caf1ed3b012e862c5bf23507aceb935c0cffd)

## [1.0](https://github.com/RCIC-UCI-Public/biotools-admix/compare/0.9...1.0) - 2021-06-10

### Commits

- rm scanpy. It requires building a few more packages [`267a1af`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/267a1af2b0353240c70d8caf45c50802d5d22a17)
- Add bowtie2 version 2.4.4 - address ticket #5523 [`adfa49d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/adfa49d8b97aea066b570de9cad691a1b61940a8)
- compile from source. Add lz4-static  as dependency for build [`4bc23ce`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4bc23ce1f94d65a20624180e68cbfb545c81b970)
- add sambamba, binary distro. [`3e7ae40`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3e7ae40843b94248cfed0f3ee376b99058fdb3e9)
- add fastmitocalc, beware, its perl [`d11bfa8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d11bfa823e370d46d275fa104568180bbac88c51)
- add verison to display info [`848af38`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/848af38dec172a81e72c52a4c376949b2a9802ed)
- add latest star [`76eb8b6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/76eb8b66013810c8f8a978354a3c478b2b389c53)
- rewrite prereq and requires to inherit from build modules [`018f442`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/018f4425103832f5a93557bbe141ff6bfc408ed2)
- use short  and long names for easier module handling [`d213b4e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d213b4e2db7d63f76f4087138e1f426db35f910e)
- use full name for rmp name, use 'trinity' [`5236e6a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5236e6aea12c943ffa65a676b7167bf569870238)
- add bcftools, samtools. Remove pilon module from build [`33152d7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/33152d76aa3680c0459337c035f9e9d8d6225f52)
- move edirect to bootstrap [`af23170`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/af23170cf6bd3b375e05367443ffd45c98cb753c)
- add bamtools to google drive [`b12689d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b12689dc42aa3b31e68eee0511de04391946141d)
- was missing [`ba00f90`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ba00f90db4fee903885b808604933ab2a228a3c5)
- rearragen names so that all names from different [`ad885bb`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ad885bb439c7c15c9bae0fa27210a4d8f0fb8d3c)
- add missing relion and its module [`a4403c7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a4403c7f9d96c177427ac55978b589ba8dfa83e6)
- rm commented lines [`556ea33`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/556ea33fa7243c09de409368c6523ce78b8d014c)
- rm old versions, sort [`0fec7be`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0fec7befb7ddadce020dd33f84e4dfa1e3a7ed10)
- use bind mount to build libraries "in destdir" [`f7a12a0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f7a12a0eae47a5224f89123c6ec4d95cba84cef6)
- add PERL5LIB, bump release. [`3c7e239`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3c7e239689e160c9df954d24d070695d337aad1c)
- add  lib flag to configure otherwisae perl files will fail on execution [`792477f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/792477f830d7531ad16827d933f94700aee260ef)

## 0.9 - 2021-03-12

### Commits

- sync cufflinks/cufflinks-module release [`b76b130`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b76b13020aca85e0a66adc28d40ab27a1e0e494b)
- keep requires for consistency. [`b9aabbf`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b9aabbffa706132b60f4b87dd33b3c76eefd2970)
- add Text::Soundex prereq for repeatmasker [`b078318`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b0783185fbe4c36141ae3b69ff88c75547c8e1e5)
- add util to PATH [`c57a851`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c57a851a1ada1b0828d40c3149acc45e730480a3)
- fix typo in prereq [`c56cf68`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c56cf68a11d45676210740d7caa673447ad7c065)
- Cufflinks needs and archaic version of boost (version 1.50). Modify build to utilize it. [`e607ff7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e607ff784830d382d3fce4a6be3b70c16baa93e3)
- Release bump on deeptools. Add full description for Trinity [`0530b75`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0530b7534fdf7c65c80a5181d3490784712878f3)
- fix include for bowtie [`7b2fde5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7b2fde5873a9e3482fab8ad9ec71d431c648ec8f)
- add sratools latest from binary distro [`6ae190b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6ae190b196c460c493815833c3962874ec554352)
- add TrimGalore [`573a4a5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/573a4a51a5ba23142ad1445bad9eb41f52ffd3e3)
- add picard tools v 2.24.1. Note changed and crazy build [`e783735`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e7837359541ca6948308b446ab2b7962e36a3a85)
- add gffread. See README for gclib [`5c53ba0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5c53ba0f08232d1b06a99fbc9a96d5d50b8b2e9e)
- add subread [`99ef983`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/99ef983c86dad5a206a9bc66568a7f84e071b831)
- add dependencies for deeptools [`cec31e2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cec31e2ff4eeeec430fadbcdd940fb073e891178)
- build bowtie2 with python2, needed by tophat. [`568ac0c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/568ac0c50dbea84eab4bf9d51d71e6d530a3ef5c)
- Add deeptools [`f417ee1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f417ee12a8a25e8b9f1d519e88aed282205edc8f)
- Adjust PATH for trinity to include utils/plugins and top-level app directory [`46ceda9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/46ceda900f6801f1b57f75f60dd0b5f61f36b86b)
- Fix bowtie module to properly list intel-tbb module. [`0bc7770`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0bc77700bdb51a1d7ec9ed38f79170a2197ccd1c)
- Starting the fun of the prereqs for Trinity.  Looking at their dockerfile, jellyfish is needed. [`5e36848`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5e36848df6b8307ec4148ed1a23270f912adf7e9)
- obsolete, keep track in admixbuilder/next-lab-software [`436e174`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/436e174f5c0f11ba504a0a8811207a8bafc0a635)
- fix lib64 location, add prereq jsoncpp [`b73e2c7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b73e2c7fce157c1489834b49e7054df84f193cd2)
- add jsoncpp library. Need for bamtools [`5133597`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/51335972bfd17e23699cf6fce83752c94294a517)
- update version to the 4.1.9.0 [`0b01f92`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0b01f92c6bbc28bbb76b4c48d29ed64726c664af)
- use zip source which has prebuilt version. tar.gz is not the same. [`f59e624`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f59e624d6a755bf31997a63570aa5b227ecceb9c)
- specify compiler for mpi module. [`3ec0d0e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3ec0d0e67e9241b36999cc78dad13377956c28c6)
- add meme [`8f51d0c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8f51d0cad195538d7a8efbe2ed2600859bc583b5)
- fix indentation [`f5890a0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f5890a088579f4712fca585089f14180fb6b64f3)
- rm fossils. Move compile/use info to README [`ca2e343`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ca2e343d18fd3bf8d90958b149e1315673940d39)
- bump release. [`901ddbe`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/901ddbe375ad5d9a9736314bd937eec733e3baae)
- add macs module [`e746145`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e74614528fe8bae176b504d58753ffa56a1fee37)
- use standard git repo [`612a037`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/612a03790de08c3714040a42a6df1be81454688c)
- rm duplicate [`ff28d88`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ff28d882a4194b7eee0d5e93f2150054ce305023)
- update minimap with k8 and paftools.js [`878f908`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/878f908d70199a53b092733d6943be55ed233f42)
- add top install dir to RPM [`16ebe0e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/16ebe0e01e1b170e1e3cfaf128154f1ed4ae7fa2)
- Add bamtools [`6ae0d81`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6ae0d817e3de5304a46fa2da54bfef430d72813e)
- Bump releases of ncbi-ngs, ncbi-vdb to reflect updated configuration parameters [`56dcd84`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/56dcd847e4cf568087f4f8bf81b93299996c5fe4)
- fix paths, and libs [`b64a647`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b64a647906bd8911539f1821494a8e710a70cd16)
- fix build and bump release of vcftools/bcftools [`4c73462`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4c73462ec853016f723d042870d3071ac55d0057)
- bump release to force update [`7b8eabe`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7b8eabe4ddd34461a66ca61cbf88549c5541aaaf)
- update to new module format [`82a86d2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/82a86d22d0ee65454c2fcba65ad4b5e1c0d145d2)
- add smolydn, smoldyn-module to packages.yaml [`d4c6e50`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d4c6e5015466bd2f133e366ac2f3a650432d6095)
- Add additional requires/prereqs for unicycler-module [`e1548b0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e1548b0321ab96e01f8b085f91fb73cf1f3106a8)
- Add htslib_CPPFLAGS to CPPFLAGS in cufflinks configure. Bump release of cufflink to 2. [`61bc312`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/61bc3121c85d883ae5ed23a557116ed2ec50e9b7)
- add smoldyn to google drive [`dabcc28`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/dabcc2820a5c4270404293574317d78fb290bf88)
- add smoldyn [`ea10978`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ea10978bf95122cf4fbd54e778eddb059faa1f24)
- update release [`c188054`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c188054c6ed763be2a236ae6f6a0ecb81d047a6a)
- add GATK_HOME [`79d09ec`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/79d09ece0aade71e33024bad9b0c28ccc815041f)
- update rpm filter anchor name [`e8b651c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e8b651c791fd1b9aaed3e78e1da4a7ed1aee3c74)
- use rpm.yaml [`c1680da`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c1680da9cbc7183194f29b327fd83392e67c633f)
- Add macs [`3e2d4ae`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3e2d4ae00399d6da3908688bd86f02f666eba53e)
- Add macs [`31379d4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/31379d4c33380e1f49ea2fde9882b7f7a72f6c67)
- Add macs [`698e0b5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/698e0b5bc2f7b7794d1981507e2141ce3133a311)
- temporarily disable macs [`44fec83`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/44fec838b45d11ef812e07950c3f7a9da5733ad4)
- rm duplicates [`c9376e5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c9376e599b0ee4a421383145adbc7223bb913f5e)
- add latest packages [`a095ec8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a095ec85b13b8f7fd201f1949dd71de8ff29810e)
- consolidate stitch RPMS and dependencies [`4f875ab`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4f875ab94ac43489985bb27a50543490959e9434)
- moving to ruamel version [`3d3a809`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3d3a80977dcb8e174d4a60643efb05cb6b49e5ce)
- add gcc libiconv [`9dac2b0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9dac2b0d41a83a52f069ab34dd93303b9ccb46a9)
- add MACS to google drive. [`7b1654e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7b1654e43c87cbeb71e77637e5f5201c1f510d6d)
- macs added to versions.yaml [`3c61537`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3c61537d98da1a6cf4cd11d528103c8a5f235bdd)
- release bumps. Fix requires in samtools, cufflinks modules. Add MACS to packages.yaml [`08ab7e7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/08ab7e7bc5a3481e6aad66c60c54364b8bbabceb)
- Add cufflinks and module [`7443cc8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7443cc829c20e20b686cd506368aae96e702f617)
- update requires [`cb95fce`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cb95fce3631327a050e3fa51a651f3e8cc53ec6a)
- initial qtltools [`2b29469`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2b29469fc18861fc4da6513c699b05d43faa8923)
- Add STITCH R module [`e7fdfb8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e7fdfb80102ba9940efb7973c8db7e61a4acd501)
- bcftools needed another perl module (ExtUtils:Embed) for build [`8203514`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/82035141f97446828f2ef59186b5fe8bef155dc4)
- back out versions for perl modules that are now part of the perl admix [`ff503c5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ff503c54ecdb93ec0bbe8b7b3f322abbb0515e68)
- updates for bowtie2 to filter perl requires/provides. Unicycler build for intel-tbb [`f4b23c6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f4b23c6a9f40c2683f36c3cc267d727414c357a6)
- remove 'v' from name of rsem tarball. [`ba767ab`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ba767abde870f7c584b2f014bedb46e692510198)
- samtools-1.10 was being downloaded to the incorrect path. FIX [`0fc6d06`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0fc6d062ee6e64e02ac936d35e832ff9d60eef7c)
- update to latest available [`cc9daf7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cc9daf78e1ed19561ca8ec70fd40ec18822b005f)
- fix vendor source [`451150a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/451150a144a0ee38c450ee56a6e09eaf5e5390ff)
- rename blast as ncbi-blast to keep consistent [`0bfe7f6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0bfe7f6a7b4910e507f3f8cf4af5dcce1e05a1bc)
- put repeatmasker back to build [`b7aaf50`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b7aaf50bc5d9bef0d27df06fee3390dbe4161c98)
- update version. Fix dependency source. [`922c418`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/922c41896afcddc5eb260d1440923ddffd724f91)
- ncbi-ngs is done during bootstrap [`93150aa`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/93150aabd65a21036d8880b4f2fbd44da1fb28bc)
- fix static library name [`fcdb7ed`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fcdb7ed5d857a8afdbd28c38de82b1f63c0c0a7d)
- add source for vcftools [`02efda3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/02efda3c9c8ae6ed66362944355dedda7fc37c33)
- add mapDamage [`efcc2c7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/efcc2c7871315a4fb43daaa2be15881be692f859)
- add freebayes and vcflib (different from vcftools) [`be6a1cc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/be6a1cc04e20d96aa3dc7dc43ea8260c5e6a9f06)
- add -j 8 to make for vg. [`cd6c3fb`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cd6c3fb453e38f4afeced2bd233a83644922ddc8)
- make -j 8 to speed up build [`58601aa`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/58601aaa383d8b2265103bfe509c656250aaff98)
- rm library definition from the configure line. [`5162a96`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5162a96d013926e503865d8e99ad0f384111b65b)
- gsl version 2.5 --&gt; 2.6 [`bdb44f7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/bdb44f70acaeb00f6608377a35ea3d5b1706be6f)
- add fastqc [`c319bc3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c319bc31a6de8bfe3d46455146dfe73689012d77)
- split description into lines [`008e935`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/008e935f3fb2abb29d25a52894bb31d6d88a8c21)
- Unicycler complained if python wasn't python3 on the command line. [`eb5518e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/eb5518eb1e8a0550e06f9a1c5d480e06f5cf125f)
- add mapDamage version to versions.yaml and its checksum to metadata file. [`44f3dea`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/44f3dea1c402850a69c844a0fcf841d321cad41c)
- add vcftools to manifest [`528c0bd`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/528c0bdbdc04c97c3d5e51e4016db1cb547e5708)
- add vcftools [`6b563bf`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6b563bff1536dc4ba2b8c8a3fefe29b53f5336be)
- add packages to google drive. [`ab2090b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ab2090b0309284877f5c80d5c375a52925de7b5d)
- add latest packages to build [`6133d97`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6133d97b3e0d27f7a3ebcd57aea8bf320751dc36)
- rm comments. [`69310ec`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/69310ec268e4d9f3fe1d7cc9008f265b9f04df8a)
- add unicycler [`177d28e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/177d28e86e962b4d18d8ae5e0428fdb53aa5be49)
- use python3 [`5aed6cb`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5aed6cb3ac9e8be71355788f5dfdda5b1fed3cbf)
- use compiled perl (has Math-Random) and python3 [`73fafa3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/73fafa38c6c93511439884c5075934dec8bd3253)
- add ucsc-tools. [`b3812c0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b3812c05867f0237849c1de6d6885adf2435cc88)
- add scanpy. rm PBSuite [`25f04c9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/25f04c9fce3dcdd222edad6667a5d33c66c13e3e)
- add PBSuite, build its prereqs for python2/3 [`6d5d99d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6d5d99dd51f88ee0889cd7e94278795ae672e9a1)
- R is now 3.6.2 [`45a359f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/45a359fde4c3c44fb1134559cb87d9a7630d970d)
- add sra-tools [`dce1476`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/dce147661bdb1b9119b40ab6ad0ada4c63c3d68b)
- simplify configure line [`360a9d9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/360a9d9f50c08e0771c53f6ba00345a4919c636b)
- build prereqs for SRA tools [`73b06b9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/73b06b932849ad635ffd585568afb49e8b3ecbdf)
- need includes and libs from file-devel RPM to compile ncbi-vdb [`9c05cc5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9c05cc5af0405d2c7e4fe6fc25bedf57887d14b9)
- add RepeatMasker [`5d8c8b4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5d8c8b4d9ef0478e9573afa4f074557cb887f724)
- add a link to binary, replaces an alias [`6080056`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/60800564f7ecb6a8f90b6b2acf4f9b833b6ef84d)
- add TRF and HMMER, prereqs for RepeatMasker [`4ef95de`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4ef95deff6b52eb37573cb1175825cb0b679ecf1)
- add env [`f645416`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f645416baa4725012d8c04edd19b669e0294bc96)
- add patch to make rmblast [`7ff60f5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7ff60f51950805642593ce9a79d5eb975f86c4ec)
- rm extraneous edits [`db4076b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/db4076b941fb7ae542952890c129f59acfc8ada1)
- enable requires filter. [`4f68ba7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4f68ba73c01d5241613a41c06d4c570fa067c517)
- use proper tbb version [`f73357a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f73357a38a209435cd2280794fcf7aed430964ba)
- add transdecoder [`f314cf8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f314cf8d415d48985350dae1b489377568681c0a)
- add bowtie [`3b88e0c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3b88e0cf44a4052a330400da271535aedb95ec00)
- add star [`0b75f9f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0b75f9f6d1c536f4bb6b013675eeddd293a9b10e)
- add rsem [`b149f74`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b149f74a67024021a35b4c567b4ac73006e4b152)
- Generic top-level Makefile [`40d2e65`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/40d2e658995b5b8c8779185ba8cb9bdc327c0359)
- no modules needed for edirect [`361164d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/361164da295fe739882031c6bfee7d9e99de7982)
- updates per changes in default include files [`978ed96`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/978ed965ec2d2a29d5521dd4333c9972f946b05e)
- Additonal software sources [`6728824`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/67288240b888f8f61921b00697c8eff8cff774e0)
- vg module requires update [`ed386e2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ed386e28551e680db1dea884df8b36672fb8d3ef)
- minimap2 build. Both C (executable) and python (cython) versions [`df75283`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/df75283cc1dcefeff53bfe92a208a449ea5f3bd2)
- desired prereq for unicycler. [`a2783ff`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a2783ff7454a66fc8c7f7d45e85de5a79c857b26)
- add runtime dependency on edirect [`6c828ec`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6c828ec541610025b12d090ca70d71cc097d97ab)
- perl prereqs for edirect [`95e51e0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/95e51e055b47ec80c8a0eb7a775471f8b9ace445)
- bustools [`b2295b1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b2295b19688de59f24c69627e58d3aa925f703ec)
- nadya finished blast [`b8b4b50`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b8b4b50d9389a5e777645eb0bc32e6d7b1da999f)
- Kallisto completed [`9906f33`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9906f33fb453b997e25ff55b5387c3d909e1853d)
- add kallisto support [`59da458`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/59da4585d8c004da31670c90289c5a90cb0abc6c)
- dependencies for unicycler: pilon, racon, blast [`8e75db7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8e75db7ea919803217e96419c3995d5b533d6cf7)
- rm fossils [`d4fdbee`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d4fdbeef5f79a8bf072c7f47419552aa591d59bf)
- add pysam support [`fc56e46`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fc56e46c9721b7a1110c76d3a0426ce710ac4a8a)
- add bedtools2 and set of REQUESTED packages from the Mortazavi lab [`e745d6f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e745d6f7d484eceea059b4f0873c07a26a2c8be7)
- update SPADes version [`8b33da2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8b33da21b720eceff932c6fc34f74daf59005ebd)
- load rpm extras from file [`70735e7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/70735e7f0b581b230a29bd672039318c16db80a5)
- commited by mistake, rm [`98bf7bc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/98bf7bc6625bc6824667345f5c057519d7a126a0)
- fix vendor source and version [`ef9038a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ef9038a29dc658c78916d1ef18c1cc40c7c0ab99)
- add PATH [`1660f3c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1660f3cec19aa68609ba65d93b7bd7f5491d12bf)
- fix compile, use java 1.7.0: newer java does not work [`f03e468`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f03e46882ca055368a807cf0c80355e279e0652b)
- fix file for gatk-classic [`4f2c357`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4f2c35734e34b4f3abbf44075f27ff89f97b252a)
- add new gaatk source [`77e7207`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/77e72070961ab7bb4124de2b9a323deee5ab1861)
- Use system-defined rcic-module.yaml file [`eb8cd58`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/eb8cd58122dac88d0b3c0dbdce2aaddc067d3f11)
- Set requires for java module load in gatk [`c1fbc82`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c1fbc827affe4eee1e29ee9333596d97526ec603)
- add .gitignore [`a0cb6c3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a0cb6c3819c5b43cccd9d218fedaff56586ef05e)
- add notes for packages [`7d8463e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7d8463ef846b0afadfd2f63f52616b65c20eaa18)
- rm exec bit [`b454153`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b454153f24edc7e9f2efd2c3057d53765aad5a03)
- use packages.yaml [`40240cd`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/40240cdd02c43e0918ed5c0d7d43cce0b22a75cd)
- unify build variables via common.yaml [`5149f16`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5149f1669486e9b6ef4d54667c20d0f7d319a0a3)
- disable jar  files recompile. [`cc6c293`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cc6c293d7115c9a3e45b9ce9992d4b1009679def)
- metadata file for biotools [`996760f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/996760fc8be471ee67538878a4fac0b6af01c2e4)
- add gsl libs location; comment empty definition of CPPFLAGS [`76c7a8d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/76c7a8d4c0848bbc0afd72e1c98fb85ed0b23143)
- simplify handling of 2 versions [`46faf8c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/46faf8c4dff2cac809de896b518503216f2a4eae)
- modify gatk to use standard build. only gatk-classic works right now [`bcc8a92`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/bcc8a92de6adf0587870c0d9232794eed66e7f10)
- update samtools and tophat to new build style [`df93277`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/df932777acb93d5e77624a4fc2356b8211fc69a0)
- SPAdes is part of new build style. rename bwa-module-latest to bwa-latest-module for consistency [`4ee8eee`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4ee8eee0654660b76a2680f693e7f69f1fd72d99)
- picard tools and module to new build style [`c206cb7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c206cb7954df5bfb467c024faae4229e343e6224)
- salmon, htslib, bwa-latest update to new build standard [`eef6ad0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/eef6ad0d97b21aab6c0f68975ec1ceda357423c6)
- Update fastp to latest build structure. Separate out fastp module def [`98d9590`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/98d959003c2268bbcbc3d936b28f5691d8b58d03)
- update vg edirect to new includes [`533df82`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/533df82bf2a6ba4f2ab315779197ca3ef7516adc)
- Update bwa to use improved rcic-package and rcic-module includes [`add8c73`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/add8c730f1b04d643bd2b7a2a370230863624991)
- update bowtie2 to use versions.yaml [`8f47b08`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8f47b0847ab73c4e3f52ca6497ae6a55c0831d0e)
- bcftools updated to use gsl. [`47c5b09`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/47c5b0959b3282900af9b399fcb99c48a2e0e720)
- new path for modules.  Create packages.yaml. [`0443026`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/044302653d996b9a903342336f7dd7188584a30c)
- added vg-built libdeflate.so to installed software [`641d3ab`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/641d3ab653e55cbd831589b065a279ef3c8b6b3c)
- Add salmon to modules.build [`8400a07`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8400a07e0a56046fc093494d798e0642751dd8c8)
- Salmon build [`cc49060`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cc490606776c55178f0c819e529c5370980081bc)
- htslib needs to be bootstrapped. [`7e2427d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7e2427d828f7a712b6b3d335266c58fa49f74bff)
- .gitignore [`692e083`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/692e0832ec78d3cb2463439645ac29d4151bc4bb)
- Add VG (variant graph) [`b6c3b2a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b6c3b2a79f208c409e3fbeeb2e7cc427e957e56e)
- add to the default list of packages to be built [`fd6ba0f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fd6ba0fd56f53058f368af95ef91129e9c3fbe0a)
- versions. Set requires for picard-tools and gatk-classic to depend on jre =  1.7.0 [`2359f25`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2359f25b68bb455dfc7f8ee577f3bf8bb286e82a)
- Create an RPM for entrez direct (edirect) [`6d90de5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6d90de5e52cf26dd6cb532dd4a4b3c6c50540d94)
- gatk and gatk-classic [`23f8472`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/23f8472f0ff0e29b70eb38b0d1d3f89d8434aaf4)
- Use system version of rpm.extras.yaml [`6e289d1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6e289d1b1600ecac4e6b7d158e2ba920d4fee92a)
- Update versions for gatk. Use system version of rpm.extras.filter.yaml [`57bd2dd`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/57bd2dd265c774cd8e7d16f1c7dd1784a959fa7b)
- Add fastp [`4256f58`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4256f5829b01967895bddafcd599de6e6dc995a8)
- add picard tools [`3a5883b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3a5883b47d93e509d458a7bc7ffd301b54e523af)
- Filter requires for bowtie2 [`8e448b2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8e448b20bb34d5d51c89bfd5a5a8306646c6c8f2)
- Add gatk.  Fixups bowtie2 [`70bd5a8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/70bd5a80986cb3f17a6561767b2dc60d8d9e5fca)
- Tophat/bowtie2 now building [`9be2f34`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9be2f34ad5c21338fab9774ce7579f5808833fb4)
- Top-level makefile [`4c65034`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4c6503437fa1536b7f9854b2f83db82ea9593de7)
- add bcftools. [`9494f3a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9494f3adfb302899b39c444e5493f279e808e519)
- Build samtools with dependency in htslib. [`2e850bf`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2e850bf922aca5bb9237c9fe5fee9663524e0d22)
- Too complicated for htslib RPM foo. But works (for now). [`fa55e9c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fa55e9c5b34872ea906ca26167c47a6a6c7fc8c7)
- Add htslib [`4e4cf7a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4e4cf7aa75cce3d46d591f81caf0effc8a88e8f8)
- Add bwa. Modify SPAdes naming [`6e1e058`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6e1e058a55aa2e959ea3899dc4e5d7e99f51a5c8)
- First version with SPAdes packaging [`960d474`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/960d474bf173dec03ca38ba810f7b53e30c52829)
- Initial commit [`998227a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/998227ab67107ee89c700514ceeaf6223907a617)

<!-- auto-changelog-above -->
# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).

## [Unreleased](https://github.com/RCIC-UCI-Public/biotools-admix/compare/2.0...HEAD)

## [2.0](https://github.com/RCIC-UCI-Public/biotools-admix/compare/1.0...2.0) - 2023-07-14

### Commits

- disable sra-tools prior to v.3. Bad usage for version 2.* [`2442a44`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2442a444a63332768040cb419906525916f4ba80)
- disable unused racon, prereq for unused unicycler [`0eb08c3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0eb08c396b59e60d717c10dbcefec1e82809e568)
- ble older unused pilon, minimap2, unicycler, meme [`040bd26`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/040bd26d64f29cdefbb241d5056aa4aa6b967b0c)
- disable bamtools and its prereq jsoncpp, unused [`4ce768f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4ce768ff56c96402f0de74f69618fb97c1a68439)
- no cufflinks [`9909089`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/990908986cf057fb7f18dfbac2f1b587df71b127)
- deal with annobin configuration [`009ae2c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/009ae2c0b7084c12e2b49b3410cdc5c8500fd4db)
- configure for jellyfish.yaml fails with possibly undefined macro: AC_LIB_RPATH [`70ddba8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/70ddba88337f6f30bc0cf52fee4c93a05f5a1d24)
- Updated version of bowtie2 [`8f11cd8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8f11cd8533c87b5e75337b7d9f8c24dfb9d51f28)
- add sra-tools 3.0.0. Has improved fasterq-dump [`95b1c40`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/95b1c409416d8e369c0cd3d3740c17f91c5a007f)
- separation of options into --root and --prefix disables building of underlying [`4a9ee52`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4a9ee524fac8d4b3f98f07b27f464077071aa41c)
- update to picard-tools was on opt1  branch and unmerged. [`545ec6a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/545ec6a632b1b2edf979e18169adef08ccab9904)
- comment out unused rrbgen [`26a8d70`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/26a8d706aa59c6e96a1c018ffb9db429dd45f18e)
- comment out rrbgen, it is a prereq for unused stitch [`5f6c323`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5f6c323218a35062c71a84aa866dccef4dffbc8d)
- Don't bootstrap in set-py2. It uses the same bootstrapped versions as the base set [`937a7ec`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/937a7ecd336ccb13bd86bf00f6edf7b5a7146fa0)
- set bootstrap: to empty in set-latest. [`d92edfc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d92edfc6988faf05e7b0044b24ff8c79cb5fcf70)
- account for difference in tarball naming of older and newer versions of star [`fe8c65d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fe8c65d1a5b192bb5bfebe5c7b0743a0949bcf1f)
- Fixed upload of libdeflate in google drive ... new hash. [`2fff95d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2fff95d3b4f5a4407eeccbfa2be275ee2ffeb36b)
- Another configure weirding...need to unset CPPFLAGS prior to python setup.py. With is set, [`50fde2c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/50fde2c2230c20fa992717c0ff4946eaf5e80de2)
- ncbi-ngs, ncbi-vdb needed to compile hisat2 in the genomics admix [`f2656b5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f2656b5e6cbe96cd2bd3aa843ccec9e28ca760bc)
- move deeptools and cutadapt into their own app directory instead of [`1b2c2e2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1b2c2e24df9d062ab3462df123ea2c5895dc7083)
- add new sources to google drive [`daacda1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/daacda1e7207e07c4c134c402e683ad72def5afd)
- add latest packages for centos8 [`ab0a919`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ab0a9194611ffde367bb01b70e303ae5e1c7f813)
- update per changes to yaml files for source, pyversion, etc.. [`8fef622`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8fef6225d398bac3837ef10dba6f5db020b2745f)
- rm unsued minimap2 version. [`fe6b2f0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fe6b2f0d7f040b249754dea940d0a43977e8e0ea)
- rename set and version for python2 build [`12861d0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/12861d0529fe57a4cb8f67dcfe175dc0fb8c8298)
- use version and pymajor variables. this allows to use a single yaml [`81de2e0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/81de2e0b68964265a438d5febc73094b65fe60a8)
- set version and pymajor variables in version*yaml files and [`dd11d00`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/dd11d001fdabf30ea8875ab61f2cb2184fd28381)
- use default naming schema for src_tarball [`2e5d416`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2e5d416ffd72afd9efc91505023937521d6d028c)
- do not add htslib.mk htslib_vars.mk to the install directory. [`943996b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/943996b4d86f6d34e18754d611fa7de26dc925a1)
- newer distros  compilation fails with a latest (above v9 gcc) [`75e7cd0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/75e7cd0dad5849cfab12d3e6abe984967bcb2262)
- update vendor source for multiple versions, unset MAKEFLAGS in build [`851e1b8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/851e1b82521cd5721893a3a7ee30dcced5d8dd48)
- update vendor source for multiple versions [`3b7aa7b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3b7aa7b913187de972741f05c8bde2ebf2e19aa7)
- add --skip-npm option to a newer version build. [`615dbe4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/615dbe45a6755c6fe1c96b44a0c0b128e5801293)
- update vendor source pre new distro. [`8fd18a1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8fd18a17a947d3aeafd20ae164c98b1a96a4545e)
- update download link per new hierarchy. [`8b3795f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8b3795fdda8b0e09d108d8a02f418a155d859872)
- add a few more libraries to ignore in requires. [`48ce789`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/48ce78907ec25867b1759c83ae97691560282387)
- newer java openjdk do not provide "jre = " but provide "jre-" [`78b8c3c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/78b8c3c42d9e1fe616cf792035c443163dcd8d3d)
- newer version of fastp depends on 2 additioal libraries whihc come [`84bf5f6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/84bf5f674b0c51b5c79f07dac3fc2554eeb3fd36)
- cufflinks depends on boost where older version of boost had include/boost/tr1/* [`1a75575`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1a755757ee169183ef51532f635ac1b904f327c1)
- ideveloeprs do not version their distros. hae to go by year so that [`8f17f33`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8f17f33de7998fec4d6beeb01355bbac43ce93ed)
- use hash for multipel versions [`6b1d89a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6b1d89a79368845004b2b05b1f1722eaaee9a2c9)
- accommodate for patches releases [`ded211f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ded211f72cea846082f52cafab49de45c4c0cce9)
- source distro name change i late version. [`6446edf`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6446edf911318c958b9dad03cec734b86296ad3a)
- user foundaiton module with version [`858d5f7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/858d5f7abafe47f6ddb261810a888d3f546f90e1)
- one digit was flipped in checksum, fixed [`9cd8f1d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9cd8f1d5b3ef9eea4a40b27b6471991ca7b8563a)
- Add intel_tbb [`1462dc4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1462dc4b3ac507b01b4f2db3c1cfcdb05f21ad25)
- install jar file [`a3cf1f2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a3cf1f27fc4702742bd28c47752c0bc167347170)
- unset MAKEFLAGS [`d560427`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d5604273df45af33b74977912b388b4010d43c88)
- unset MAKEFLAGS [`c461cd0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c461cd01e78f55aaadd4f0f93cf8024f7e59f693)
- add src_tarball [`412940c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/412940c76ab2bb949daeed700cc8ed2df25edbd5)
- Don't build id links for minimap2  (both versions include same binary libraries) [`8c4f48e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8c4f48e8f867489864ef52e518f5153c27efbb2b)
- Remove perl requirements that are not explicit in trinity [`bbeb53e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/bbeb53e9602442e8a0d912e3f3267acff59fc9e5)
- System packages that are not available on EL8 (lz4-static, perl-Package-Constants) are not requested on EL8. [`0d08ac8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0d08ac8b8e0efed16ea2cbdd8b421a169ff1c268)
- Build cufflinks under EL8. Requires specific commit from cufflinks github. [`ed263bc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ed263bc79cbdc80dfcb7331b49217f8238326510)
- Build bcftools with annobin turned off on EL8 (and later) [`29a270e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/29a270e4f1763c4c5b32e0017df93c1d9c5cc0ee)
- Sambamba build with updated liblz4.a (built in buildtools) [`6f26821`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6f26821ac9205c590ec690590faca70449ed5128)
- add minimap2-2.23 to google drive [`42a65da`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/42a65da13b5e5126252b600932b1d46d420f6de0)
- change install path and module settings so that [`10bd6ca`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/10bd6ca9c5660080c25f6d4d5390b7425871864a)
- gsl is in site versions [`868cfe6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/868cfe60ff3c8128445a8868457c39360c61dd9c)
- use site versions [`5c29123`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5c29123d0cebaf5c724ee27da3657e61d8651c4b)
- in preparation to use common versions file [`3b0105d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3b0105d5927907d333c53406be600ce17c461153)
- add .rpmmacros [`26fcf49`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/26fcf49dba08b947154f2bc1c772fce77a65f8a2)
- delete extra definitions [`1e1b5a6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1e1b5a6913cd8a97a50d98be386aad42a8252810)
- sra tools fix typo gcc. -&gt; gcc/   in required modules [`f7d38eb`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f7d38ebd6b47560a27844718112e3f180c703a85)
- update naming schema [`801a23c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/801a23cf17d47bb597525631da3b8300a0fe2fb0)
- verify configure and its arguments [`d50a9b5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d50a9b5195172421163352a275843e9777e99e06)
- moving to set based build [`49c2dca`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/49c2dca9adb2db8899d7a7640f9ebca8ded10ff7)
- add breseq as requested in ticket #5850 [`9aa1bfc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9aa1bfcd5bb08dce9fb4b2dd612af83248f8d083)
- rm empty strings [`54e916f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/54e916f1e765b12e875ce032ff8fdf6a835b303b)
- initial cleanup for opt1 changes [`5580476`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5580476f114708cb523e74a18e5d59aa0e808ba1)
- add filters for perl and dependency on perl. [`1d6caf1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1d6caf1ed3b012e862c5bf23507aceb935c0cffd)

## [1.0](https://github.com/RCIC-UCI-Public/biotools-admix/compare/0.9...1.0) - 2021-06-10

### Commits

- rm scanpy. It requires building a few more packages [`267a1af`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/267a1af2b0353240c70d8caf45c50802d5d22a17)
- Add bowtie2 version 2.4.4 - address ticket #5523 [`adfa49d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/adfa49d8b97aea066b570de9cad691a1b61940a8)
- compile from source. Add lz4-static  as dependency for build [`4bc23ce`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4bc23ce1f94d65a20624180e68cbfb545c81b970)
- add sambamba, binary distro. [`3e7ae40`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3e7ae40843b94248cfed0f3ee376b99058fdb3e9)
- add fastmitocalc, beware, its perl [`d11bfa8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d11bfa823e370d46d275fa104568180bbac88c51)
- add verison to display info [`848af38`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/848af38dec172a81e72c52a4c376949b2a9802ed)
- add latest star [`76eb8b6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/76eb8b66013810c8f8a978354a3c478b2b389c53)
- rewrite prereq and requires to inherit from build modules [`018f442`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/018f4425103832f5a93557bbe141ff6bfc408ed2)
- use short  and long names for easier module handling [`d213b4e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d213b4e2db7d63f76f4087138e1f426db35f910e)
- use full name for rmp name, use 'trinity' [`5236e6a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5236e6aea12c943ffa65a676b7167bf569870238)
- add bcftools, samtools. Remove pilon module from build [`33152d7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/33152d76aa3680c0459337c035f9e9d8d6225f52)
- move edirect to bootstrap [`af23170`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/af23170cf6bd3b375e05367443ffd45c98cb753c)
- add bamtools to google drive [`b12689d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b12689dc42aa3b31e68eee0511de04391946141d)
- was missing [`ba00f90`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ba00f90db4fee903885b808604933ab2a228a3c5)
- rearragen names so that all names from different [`ad885bb`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ad885bb439c7c15c9bae0fa27210a4d8f0fb8d3c)
- add missing relion and its module [`a4403c7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a4403c7f9d96c177427ac55978b589ba8dfa83e6)
- rm commented lines [`556ea33`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/556ea33fa7243c09de409368c6523ce78b8d014c)
- rm old versions, sort [`0fec7be`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0fec7befb7ddadce020dd33f84e4dfa1e3a7ed10)
- use bind mount to build libraries "in destdir" [`f7a12a0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f7a12a0eae47a5224f89123c6ec4d95cba84cef6)
- add PERL5LIB, bump release. [`3c7e239`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3c7e239689e160c9df954d24d070695d337aad1c)
- add  lib flag to configure otherwisae perl files will fail on execution [`792477f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/792477f830d7531ad16827d933f94700aee260ef)

## 0.9 - 2021-03-12

### Commits

- sync cufflinks/cufflinks-module release [`b76b130`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b76b13020aca85e0a66adc28d40ab27a1e0e494b)
- keep requires for consistency. [`b9aabbf`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b9aabbffa706132b60f4b87dd33b3c76eefd2970)
- add Text::Soundex prereq for repeatmasker [`b078318`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b0783185fbe4c36141ae3b69ff88c75547c8e1e5)
- add util to PATH [`c57a851`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c57a851a1ada1b0828d40c3149acc45e730480a3)
- fix typo in prereq [`c56cf68`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c56cf68a11d45676210740d7caa673447ad7c065)
- Cufflinks needs and archaic version of boost (version 1.50). Modify build to utilize it. [`e607ff7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e607ff784830d382d3fce4a6be3b70c16baa93e3)
- Release bump on deeptools. Add full description for Trinity [`0530b75`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0530b7534fdf7c65c80a5181d3490784712878f3)
- fix include for bowtie [`7b2fde5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7b2fde5873a9e3482fab8ad9ec71d431c648ec8f)
- add sratools latest from binary distro [`6ae190b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6ae190b196c460c493815833c3962874ec554352)
- add TrimGalore [`573a4a5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/573a4a51a5ba23142ad1445bad9eb41f52ffd3e3)
- add picard tools v 2.24.1. Note changed and crazy build [`e783735`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e7837359541ca6948308b446ab2b7962e36a3a85)
- add gffread. See README for gclib [`5c53ba0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5c53ba0f08232d1b06a99fbc9a96d5d50b8b2e9e)
- add subread [`99ef983`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/99ef983c86dad5a206a9bc66568a7f84e071b831)
- add dependencies for deeptools [`cec31e2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cec31e2ff4eeeec430fadbcdd940fb073e891178)
- build bowtie2 with python2, needed by tophat. [`568ac0c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/568ac0c50dbea84eab4bf9d51d71e6d530a3ef5c)
- Add deeptools [`f417ee1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f417ee12a8a25e8b9f1d519e88aed282205edc8f)
- Adjust PATH for trinity to include utils/plugins and top-level app directory [`46ceda9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/46ceda900f6801f1b57f75f60dd0b5f61f36b86b)
- Fix bowtie module to properly list intel-tbb module. [`0bc7770`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0bc77700bdb51a1d7ec9ed38f79170a2197ccd1c)
- Starting the fun of the prereqs for Trinity.  Looking at their dockerfile, jellyfish is needed. [`5e36848`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5e36848df6b8307ec4148ed1a23270f912adf7e9)
- obsolete, keep track in admixbuilder/next-lab-software [`436e174`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/436e174f5c0f11ba504a0a8811207a8bafc0a635)
- fix lib64 location, add prereq jsoncpp [`b73e2c7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b73e2c7fce157c1489834b49e7054df84f193cd2)
- add jsoncpp library. Need for bamtools [`5133597`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/51335972bfd17e23699cf6fce83752c94294a517)
- update version to the 4.1.9.0 [`0b01f92`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0b01f92c6bbc28bbb76b4c48d29ed64726c664af)
- use zip source which has prebuilt version. tar.gz is not the same. [`f59e624`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f59e624d6a755bf31997a63570aa5b227ecceb9c)
- specify compiler for mpi module. [`3ec0d0e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3ec0d0e67e9241b36999cc78dad13377956c28c6)
- add meme [`8f51d0c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8f51d0cad195538d7a8efbe2ed2600859bc583b5)
- fix indentation [`f5890a0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f5890a088579f4712fca585089f14180fb6b64f3)
- rm fossils. Move compile/use info to README [`ca2e343`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ca2e343d18fd3bf8d90958b149e1315673940d39)
- bump release. [`901ddbe`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/901ddbe375ad5d9a9736314bd937eec733e3baae)
- add macs module [`e746145`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e74614528fe8bae176b504d58753ffa56a1fee37)
- use standard git repo [`612a037`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/612a03790de08c3714040a42a6df1be81454688c)
- rm duplicate [`ff28d88`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ff28d882a4194b7eee0d5e93f2150054ce305023)
- update minimap with k8 and paftools.js [`878f908`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/878f908d70199a53b092733d6943be55ed233f42)
- add top install dir to RPM [`16ebe0e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/16ebe0e01e1b170e1e3cfaf128154f1ed4ae7fa2)
- Add bamtools [`6ae0d81`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6ae0d817e3de5304a46fa2da54bfef430d72813e)
- Bump releases of ncbi-ngs, ncbi-vdb to reflect updated configuration parameters [`56dcd84`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/56dcd847e4cf568087f4f8bf81b93299996c5fe4)
- fix paths, and libs [`b64a647`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b64a647906bd8911539f1821494a8e710a70cd16)
- fix build and bump release of vcftools/bcftools [`4c73462`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4c73462ec853016f723d042870d3071ac55d0057)
- bump release to force update [`7b8eabe`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7b8eabe4ddd34461a66ca61cbf88549c5541aaaf)
- update to new module format [`82a86d2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/82a86d22d0ee65454c2fcba65ad4b5e1c0d145d2)
- add smolydn, smoldyn-module to packages.yaml [`d4c6e50`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d4c6e5015466bd2f133e366ac2f3a650432d6095)
- Add additional requires/prereqs for unicycler-module [`e1548b0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e1548b0321ab96e01f8b085f91fb73cf1f3106a8)
- Add htslib_CPPFLAGS to CPPFLAGS in cufflinks configure. Bump release of cufflink to 2. [`61bc312`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/61bc3121c85d883ae5ed23a557116ed2ec50e9b7)
- add smoldyn to google drive [`dabcc28`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/dabcc2820a5c4270404293574317d78fb290bf88)
- add smoldyn [`ea10978`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ea10978bf95122cf4fbd54e778eddb059faa1f24)
- update release [`c188054`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c188054c6ed763be2a236ae6f6a0ecb81d047a6a)
- add GATK_HOME [`79d09ec`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/79d09ece0aade71e33024bad9b0c28ccc815041f)
- update rpm filter anchor name [`e8b651c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e8b651c791fd1b9aaed3e78e1da4a7ed1aee3c74)
- use rpm.yaml [`c1680da`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c1680da9cbc7183194f29b327fd83392e67c633f)
- Add macs [`3e2d4ae`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3e2d4ae00399d6da3908688bd86f02f666eba53e)
- Add macs [`31379d4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/31379d4c33380e1f49ea2fde9882b7f7a72f6c67)
- Add macs [`698e0b5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/698e0b5bc2f7b7794d1981507e2141ce3133a311)
- temporarily disable macs [`44fec83`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/44fec838b45d11ef812e07950c3f7a9da5733ad4)
- rm duplicates [`c9376e5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c9376e599b0ee4a421383145adbc7223bb913f5e)
- add latest packages [`a095ec8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a095ec85b13b8f7fd201f1949dd71de8ff29810e)
- consolidate stitch RPMS and dependencies [`4f875ab`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4f875ab94ac43489985bb27a50543490959e9434)
- moving to ruamel version [`3d3a809`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3d3a80977dcb8e174d4a60643efb05cb6b49e5ce)
- add gcc libiconv [`9dac2b0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9dac2b0d41a83a52f069ab34dd93303b9ccb46a9)
- add MACS to google drive. [`7b1654e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7b1654e43c87cbeb71e77637e5f5201c1f510d6d)
- macs added to versions.yaml [`3c61537`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3c61537d98da1a6cf4cd11d528103c8a5f235bdd)
- release bumps. Fix requires in samtools, cufflinks modules. Add MACS to packages.yaml [`08ab7e7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/08ab7e7bc5a3481e6aad66c60c54364b8bbabceb)
- Add cufflinks and module [`7443cc8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7443cc829c20e20b686cd506368aae96e702f617)
- update requires [`cb95fce`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cb95fce3631327a050e3fa51a651f3e8cc53ec6a)
- initial qtltools [`2b29469`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2b29469fc18861fc4da6513c699b05d43faa8923)
- Add STITCH R module [`e7fdfb8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e7fdfb80102ba9940efb7973c8db7e61a4acd501)
- bcftools needed another perl module (ExtUtils:Embed) for build [`8203514`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/82035141f97446828f2ef59186b5fe8bef155dc4)
- back out versions for perl modules that are now part of the perl admix [`ff503c5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ff503c54ecdb93ec0bbe8b7b3f322abbb0515e68)
- updates for bowtie2 to filter perl requires/provides. Unicycler build for intel-tbb [`f4b23c6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f4b23c6a9f40c2683f36c3cc267d727414c357a6)
- remove 'v' from name of rsem tarball. [`ba767ab`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ba767abde870f7c584b2f014bedb46e692510198)
- samtools-1.10 was being downloaded to the incorrect path. FIX [`0fc6d06`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0fc6d062ee6e64e02ac936d35e832ff9d60eef7c)
- update to latest available [`cc9daf7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cc9daf78e1ed19561ca8ec70fd40ec18822b005f)
- fix vendor source [`451150a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/451150a144a0ee38c450ee56a6e09eaf5e5390ff)
- rename blast as ncbi-blast to keep consistent [`0bfe7f6`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0bfe7f6a7b4910e507f3f8cf4af5dcce1e05a1bc)
- put repeatmasker back to build [`b7aaf50`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b7aaf50bc5d9bef0d27df06fee3390dbe4161c98)
- update version. Fix dependency source. [`922c418`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/922c41896afcddc5eb260d1440923ddffd724f91)
- ncbi-ngs is done during bootstrap [`93150aa`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/93150aabd65a21036d8880b4f2fbd44da1fb28bc)
- fix static library name [`fcdb7ed`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fcdb7ed5d857a8afdbd28c38de82b1f63c0c0a7d)
- add source for vcftools [`02efda3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/02efda3c9c8ae6ed66362944355dedda7fc37c33)
- add mapDamage [`efcc2c7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/efcc2c7871315a4fb43daaa2be15881be692f859)
- add freebayes and vcflib (different from vcftools) [`be6a1cc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/be6a1cc04e20d96aa3dc7dc43ea8260c5e6a9f06)
- add -j 8 to make for vg. [`cd6c3fb`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cd6c3fb453e38f4afeced2bd233a83644922ddc8)
- make -j 8 to speed up build [`58601aa`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/58601aaa383d8b2265103bfe509c656250aaff98)
- rm library definition from the configure line. [`5162a96`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5162a96d013926e503865d8e99ad0f384111b65b)
- gsl version 2.5 --&gt; 2.6 [`bdb44f7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/bdb44f70acaeb00f6608377a35ea3d5b1706be6f)
- add fastqc [`c319bc3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c319bc31a6de8bfe3d46455146dfe73689012d77)
- split description into lines [`008e935`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/008e935f3fb2abb29d25a52894bb31d6d88a8c21)
- Unicycler complained if python wasn't python3 on the command line. [`eb5518e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/eb5518eb1e8a0550e06f9a1c5d480e06f5cf125f)
- add mapDamage version to versions.yaml and its checksum to metadata file. [`44f3dea`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/44f3dea1c402850a69c844a0fcf841d321cad41c)
- add vcftools to manifest [`528c0bd`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/528c0bdbdc04c97c3d5e51e4016db1cb547e5708)
- add vcftools [`6b563bf`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6b563bff1536dc4ba2b8c8a3fefe29b53f5336be)
- add packages to google drive. [`ab2090b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ab2090b0309284877f5c80d5c375a52925de7b5d)
- add latest packages to build [`6133d97`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6133d97b3e0d27f7a3ebcd57aea8bf320751dc36)
- rm comments. [`69310ec`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/69310ec268e4d9f3fe1d7cc9008f265b9f04df8a)
- add unicycler [`177d28e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/177d28e86e962b4d18d8ae5e0428fdb53aa5be49)
- use python3 [`5aed6cb`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5aed6cb3ac9e8be71355788f5dfdda5b1fed3cbf)
- use compiled perl (has Math-Random) and python3 [`73fafa3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/73fafa38c6c93511439884c5075934dec8bd3253)
- add ucsc-tools. [`b3812c0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b3812c05867f0237849c1de6d6885adf2435cc88)
- add scanpy. rm PBSuite [`25f04c9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/25f04c9fce3dcdd222edad6667a5d33c66c13e3e)
- add PBSuite, build its prereqs for python2/3 [`6d5d99d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6d5d99dd51f88ee0889cd7e94278795ae672e9a1)
- R is now 3.6.2 [`45a359f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/45a359fde4c3c44fb1134559cb87d9a7630d970d)
- add sra-tools [`dce1476`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/dce147661bdb1b9119b40ab6ad0ada4c63c3d68b)
- simplify configure line [`360a9d9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/360a9d9f50c08e0771c53f6ba00345a4919c636b)
- build prereqs for SRA tools [`73b06b9`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/73b06b932849ad635ffd585568afb49e8b3ecbdf)
- need includes and libs from file-devel RPM to compile ncbi-vdb [`9c05cc5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9c05cc5af0405d2c7e4fe6fc25bedf57887d14b9)
- add RepeatMasker [`5d8c8b4`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5d8c8b4d9ef0478e9573afa4f074557cb887f724)
- add a link to binary, replaces an alias [`6080056`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/60800564f7ecb6a8f90b6b2acf4f9b833b6ef84d)
- add TRF and HMMER, prereqs for RepeatMasker [`4ef95de`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4ef95deff6b52eb37573cb1175825cb0b679ecf1)
- add env [`f645416`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f645416baa4725012d8c04edd19b669e0294bc96)
- add patch to make rmblast [`7ff60f5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7ff60f51950805642593ce9a79d5eb975f86c4ec)
- rm extraneous edits [`db4076b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/db4076b941fb7ae542952890c129f59acfc8ada1)
- enable requires filter. [`4f68ba7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4f68ba73c01d5241613a41c06d4c570fa067c517)
- use proper tbb version [`f73357a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f73357a38a209435cd2280794fcf7aed430964ba)
- add transdecoder [`f314cf8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f314cf8d415d48985350dae1b489377568681c0a)
- add bowtie [`3b88e0c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3b88e0cf44a4052a330400da271535aedb95ec00)
- add star [`0b75f9f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/0b75f9f6d1c536f4bb6b013675eeddd293a9b10e)
- add rsem [`b149f74`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b149f74a67024021a35b4c567b4ac73006e4b152)
- Generic top-level Makefile [`40d2e65`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/40d2e658995b5b8c8779185ba8cb9bdc327c0359)
- no modules needed for edirect [`361164d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/361164da295fe739882031c6bfee7d9e99de7982)
- updates per changes in default include files [`978ed96`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/978ed965ec2d2a29d5521dd4333c9972f946b05e)
- Additonal software sources [`6728824`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/67288240b888f8f61921b00697c8eff8cff774e0)
- vg module requires update [`ed386e2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ed386e28551e680db1dea884df8b36672fb8d3ef)
- minimap2 build. Both C (executable) and python (cython) versions [`df75283`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/df75283cc1dcefeff53bfe92a208a449ea5f3bd2)
- desired prereq for unicycler. [`a2783ff`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a2783ff7454a66fc8c7f7d45e85de5a79c857b26)
- add runtime dependency on edirect [`6c828ec`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6c828ec541610025b12d090ca70d71cc097d97ab)
- perl prereqs for edirect [`95e51e0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/95e51e055b47ec80c8a0eb7a775471f8b9ace445)
- bustools [`b2295b1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b2295b19688de59f24c69627e58d3aa925f703ec)
- nadya finished blast [`b8b4b50`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b8b4b50d9389a5e777645eb0bc32e6d7b1da999f)
- Kallisto completed [`9906f33`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9906f33fb453b997e25ff55b5387c3d909e1853d)
- add kallisto support [`59da458`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/59da4585d8c004da31670c90289c5a90cb0abc6c)
- dependencies for unicycler: pilon, racon, blast [`8e75db7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8e75db7ea919803217e96419c3995d5b533d6cf7)
- rm fossils [`d4fdbee`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/d4fdbeef5f79a8bf072c7f47419552aa591d59bf)
- add pysam support [`fc56e46`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fc56e46c9721b7a1110c76d3a0426ce710ac4a8a)
- add bedtools2 and set of REQUESTED packages from the Mortazavi lab [`e745d6f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/e745d6f7d484eceea059b4f0873c07a26a2c8be7)
- update SPADes version [`8b33da2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8b33da21b720eceff932c6fc34f74daf59005ebd)
- load rpm extras from file [`70735e7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/70735e7f0b581b230a29bd672039318c16db80a5)
- commited by mistake, rm [`98bf7bc`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/98bf7bc6625bc6824667345f5c057519d7a126a0)
- fix vendor source and version [`ef9038a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/ef9038a29dc658c78916d1ef18c1cc40c7c0ab99)
- add PATH [`1660f3c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/1660f3cec19aa68609ba65d93b7bd7f5491d12bf)
- fix compile, use java 1.7.0: newer java does not work [`f03e468`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/f03e46882ca055368a807cf0c80355e279e0652b)
- fix file for gatk-classic [`4f2c357`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4f2c35734e34b4f3abbf44075f27ff89f97b252a)
- add new gaatk source [`77e7207`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/77e72070961ab7bb4124de2b9a323deee5ab1861)
- Use system-defined rcic-module.yaml file [`eb8cd58`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/eb8cd58122dac88d0b3c0dbdce2aaddc067d3f11)
- Set requires for java module load in gatk [`c1fbc82`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c1fbc827affe4eee1e29ee9333596d97526ec603)
- add .gitignore [`a0cb6c3`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/a0cb6c3819c5b43cccd9d218fedaff56586ef05e)
- add notes for packages [`7d8463e`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7d8463ef846b0afadfd2f63f52616b65c20eaa18)
- rm exec bit [`b454153`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b454153f24edc7e9f2efd2c3057d53765aad5a03)
- use packages.yaml [`40240cd`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/40240cdd02c43e0918ed5c0d7d43cce0b22a75cd)
- unify build variables via common.yaml [`5149f16`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/5149f1669486e9b6ef4d54667c20d0f7d319a0a3)
- disable jar  files recompile. [`cc6c293`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cc6c293d7115c9a3e45b9ce9992d4b1009679def)
- metadata file for biotools [`996760f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/996760fc8be471ee67538878a4fac0b6af01c2e4)
- add gsl libs location; comment empty definition of CPPFLAGS [`76c7a8d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/76c7a8d4c0848bbc0afd72e1c98fb85ed0b23143)
- simplify handling of 2 versions [`46faf8c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/46faf8c4dff2cac809de896b518503216f2a4eae)
- modify gatk to use standard build. only gatk-classic works right now [`bcc8a92`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/bcc8a92de6adf0587870c0d9232794eed66e7f10)
- update samtools and tophat to new build style [`df93277`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/df932777acb93d5e77624a4fc2356b8211fc69a0)
- SPAdes is part of new build style. rename bwa-module-latest to bwa-latest-module for consistency [`4ee8eee`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4ee8eee0654660b76a2680f693e7f69f1fd72d99)
- picard tools and module to new build style [`c206cb7`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/c206cb7954df5bfb467c024faae4229e343e6224)
- salmon, htslib, bwa-latest update to new build standard [`eef6ad0`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/eef6ad0d97b21aab6c0f68975ec1ceda357423c6)
- Update fastp to latest build structure. Separate out fastp module def [`98d9590`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/98d959003c2268bbcbc3d936b28f5691d8b58d03)
- update vg edirect to new includes [`533df82`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/533df82bf2a6ba4f2ab315779197ca3ef7516adc)
- Update bwa to use improved rcic-package and rcic-module includes [`add8c73`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/add8c730f1b04d643bd2b7a2a370230863624991)
- update bowtie2 to use versions.yaml [`8f47b08`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8f47b0847ab73c4e3f52ca6497ae6a55c0831d0e)
- bcftools updated to use gsl. [`47c5b09`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/47c5b0959b3282900af9b399fcb99c48a2e0e720)
- new path for modules.  Create packages.yaml. [`0443026`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/044302653d996b9a903342336f7dd7188584a30c)
- added vg-built libdeflate.so to installed software [`641d3ab`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/641d3ab653e55cbd831589b065a279ef3c8b6b3c)
- Add salmon to modules.build [`8400a07`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8400a07e0a56046fc093494d798e0642751dd8c8)
- Salmon build [`cc49060`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/cc490606776c55178f0c819e529c5370980081bc)
- htslib needs to be bootstrapped. [`7e2427d`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/7e2427d828f7a712b6b3d335266c58fa49f74bff)
- .gitignore [`692e083`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/692e0832ec78d3cb2463439645ac29d4151bc4bb)
- Add VG (variant graph) [`b6c3b2a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/b6c3b2a79f208c409e3fbeeb2e7cc427e957e56e)
- add to the default list of packages to be built [`fd6ba0f`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fd6ba0fd56f53058f368af95ef91129e9c3fbe0a)
- versions. Set requires for picard-tools and gatk-classic to depend on jre =  1.7.0 [`2359f25`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2359f25b68bb455dfc7f8ee577f3bf8bb286e82a)
- Create an RPM for entrez direct (edirect) [`6d90de5`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6d90de5e52cf26dd6cb532dd4a4b3c6c50540d94)
- gatk and gatk-classic [`23f8472`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/23f8472f0ff0e29b70eb38b0d1d3f89d8434aaf4)
- Use system version of rpm.extras.yaml [`6e289d1`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6e289d1b1600ecac4e6b7d158e2ba920d4fee92a)
- Update versions for gatk. Use system version of rpm.extras.filter.yaml [`57bd2dd`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/57bd2dd265c774cd8e7d16f1c7dd1784a959fa7b)
- Add fastp [`4256f58`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4256f5829b01967895bddafcd599de6e6dc995a8)
- add picard tools [`3a5883b`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/3a5883b47d93e509d458a7bc7ffd301b54e523af)
- Filter requires for bowtie2 [`8e448b2`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/8e448b20bb34d5d51c89bfd5a5a8306646c6c8f2)
- Add gatk.  Fixups bowtie2 [`70bd5a8`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/70bd5a80986cb3f17a6561767b2dc60d8d9e5fca)
- Tophat/bowtie2 now building [`9be2f34`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9be2f34ad5c21338fab9774ce7579f5808833fb4)
- Top-level makefile [`4c65034`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4c6503437fa1536b7f9854b2f83db82ea9593de7)
- add bcftools. [`9494f3a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/9494f3adfb302899b39c444e5493f279e808e519)
- Build samtools with dependency in htslib. [`2e850bf`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/2e850bf922aca5bb9237c9fe5fee9663524e0d22)
- Too complicated for htslib RPM foo. But works (for now). [`fa55e9c`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/fa55e9c5b34872ea906ca26167c47a6a6c7fc8c7)
- Add htslib [`4e4cf7a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/4e4cf7aa75cce3d46d591f81caf0effc8a88e8f8)
- Add bwa. Modify SPAdes naming [`6e1e058`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/6e1e058a55aa2e959ea3899dc4e5d7e99f51a5c8)
- First version with SPAdes packaging [`960d474`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/960d474bf173dec03ca38ba810f7b53e30c52829)
- Initial commit [`998227a`](https://github.com/RCIC-UCI-Public/biotools-admix/commit/998227ab67107ee89c700514ceeaf6223907a617)
