--- bcftools-1.3.1/Makefile	2020-01-10 16:43:06.008000000 -0800
+++ bcftools-1.3.1-new/Makefile	2020-01-10 17:02:20.871000000 -0800
@@ -36,7 +36,7 @@
 TABIX  = $(HTSDIR)/tabix
 
 CC       = gcc
-CPPFLAGS =
+#CPPFLAGS =
 CFLAGS   = -g -Wall -Wc++-compat -O2
 LDFLAGS  =
 LIBS     =
@@ -57,7 +57,7 @@
 ifdef USE_GPL
     EXTRA_CPPFLAGS += -DUSE_GPL
     OBJS += polysomy.o peakfit.o
-    GSL_LIBS = -lgsl -lcblas
+    GSL_LIBS = -L$(GSL_HOME)/lib -lgsl -lgslcblas
 endif
 
 prefix      = /usr/local
