--- sambamba-0.8.0/Makefile	2020-11-30 06:46:19.000000000 -0800
+++ sambamba-0.8.0.prev/Makefile	2021-04-27 12:11:21.916524416 -0700
@@ -32,8 +32,8 @@
 
 DLIBS       = $(LIBRARY_PATH)/libphobos2-ldc.a $(LIBRARY_PATH)/libdruntime-ldc.a
 DLIBS_DEBUG = $(LIBRARY_PATH)/libphobos2-ldc-debug.a $(LIBRARY_PATH)/libdruntime-ldc-debug.a
-LIBS        = lz4/lib/liblz4.a -L-L$(LIBRARY_PATH) -L-lpthread -L-lm -L-lz
-LIBS_STATIC = $(LIBRARY_PATH)/libc.a $(DLIBS) $(LIBRARY_PATH)/liblz4.a -L-lz
+LIBS        =  /usr/lib64/liblz4.a -L-L$(LIBRARY_PATH) -L-lpthread -L-lm -L-lz
+LIBS_STATIC = $(LIBRARY_PATH)/libc.a $(DLIBS) /usr/lib64/liblz4.a  -L-lz
 SRC         = utils/ldc_version_info_.d utils/lz4.d utils/strip_bcf_header.d $(sort $(wildcard BioD/contrib/undead/*.d BioD/contrib/undead/*/*.d)) utils/version_.d $(sort $(wildcard thirdparty/*.d) $(wildcard BioD/bio/*/*.d BioD/bio/*/*/*.d BioD/bio/*/*/*/*.d BioD/bio/*/*/*/*/*.d) $(wildcard sambamba/*.d sambamba/*/*.d sambamba/*/*/*.d))
 OBJ         = $(SRC:.d=.o)
 OUT         = bin/sambamba-$(shell cat VERSION)
@@ -67,7 +67,7 @@
 
 ldc_version_info: utils/ldc_version_info_.d
 
-build-setup: ldc_version_info lz4-static
+build-setup: ldc_version_info 
 	mkdir -p bin/
 
 default debug release static: $(OUT)
